src/runtime/intern.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Python.Runtime
+{
+    static partial class InternString
+    {
+        class PyIdentifierDefinition
+        {
+            public string Name { get; private set; }
+
+            public PyIdentifierDefinition(string name)
+            {
+                Name = name;
+            }
+        }
```

Comment:
This class does not appear to be used anywhere

Reply:
Indeed, I shall remove it and push again.